SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -455,7 +437,77 @@ void InterestFactory::AddComponentQueryPairToInterestComponent(Interest& OutInte
 	OutInterest.ComponentInterestMap[ComponentId].Queries.Add(QueryToAdd);
 }
 
-void InterestFactory::GetActorUserDefinedQueries(const AActor* InActor, const QueryConstraint& LevelConstraints, TArray<SpatialGDK::Query>& OutQueries, bool bRecurseChildren) const
+TArray<Query> InterestFactory::GetUserDefinedQueries(const AActor* InActor, const QueryConstraint LevelConstraint) const
```

Comment:
Just want to point out the LevelConstraint arg is making a copy. Perhaps we want it by ref instead?

Reply:
now a reference